src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+public class Task {
+    private String name;
+    private boolean completed;
+    private String type;
+
```

Comment:
Perhaps keep the line spacings consistent, ie. there are line spacings in some files but none in others.